src/display/displayTemplateStandard.h

```diff
@@ -75,7 +72,8 @@ void printScreen()
         u8g2.print("|");
 
         if (bPID.GetKi() != 0) {
-            u8g2.print(bPID.GetKp() / bPID.GetKi(), 0);;
+            u8g2.print(bPID.GetKp() / bPID.GetKi(), 0);
+            ;
```

Comment:
not really because of this PR but this extra ; can probably be deleted ;)

Reply:
Removed.